com/hemebiotech/analytics/WriteSymptomDataToFile.java

```diff
@@ -2,7 +2,33 @@
 
 import java.io.FileWriter;
 import java.io.IOException;
-import java.util.Iterator;
 import java.util.Map;
 
-// WRITE YOUR CODE HERE
\ No newline at end of file
+
+/**.
+* Anything that will read symptom data from a source;
+* The important part is, the return value from the operation, which is a list of strings,
+* that may contain many duplications.
+* The implementation does not need to order the list.
+*/
+public class WriteSymptomDataToFile implements ISymptomWriter {
+  /** 
+   Attribute with the name of the output File.
+  */
+  private final String fileOutput = "result.out";
```

Comment:
no need for javadoc for this variables, javadoc is supposed to be existing only for method